docs/index.md

```diff
@@ -5,277 +5,162 @@ hide:
-
-### deploy server
-
-The inference server is distributed via Docker. Behind the scenes, inference will download and run the image that is appropriate for your hardware. [Here](/quickstart/docker.md#advanced-build-a-docker-container-from-scratch), you can learn more about the supported images.
-
-```bash
-inference server start
-```
-
-### run client
-
-Consume inference server predictions using the HTTP client available in the Inference SDK.
-
-```python
-from inference_sdk import InferenceHTTPClient
-
-client = InferenceHTTPClient(
-    api_url="http://localhost:9001",
-    api_key=<ROBOFLOW_API_KEY>
-)
-with client.use_model(model_id="soccer-players-5fuqs/1"):
-    predictions = client.infer("https://media.roboflow.com/inference/soccer.jpg")
-```
-
-If you're using the hosted API, change the local API URL to `https://detect.roboflow.com`. Accessing the hosted inference server and/or using any of the fine-tuned models require a `ROBOFLOW_API_KEY`. For further information, visit the 🔑 keys section.
-
-## 🎥 inference pipeline
-
-The inference pipeline is an efficient method for processing static video files and streams. Select a model, define the video source, and set a callback action. You can choose from predefined callbacks that allow you to [display results](/reference/inference/core/interfaces/stream/sinks.md#inference.core.interfaces.stream.sinks.render_boxes) on the screen or [save them to a file](/reference/inference/core/interfaces/stream/sinks.md#inference.core.interfaces.stream.sinks.VideoFileSink).
-
-```python
-from inference import InferencePipeline
-from inference.core.interfaces.stream.sinks import render_boxes
-
-pipeline = InferencePipeline.init(
-    model_id="yolov8x-1280",
-    video_reference="https://media.roboflow.com/inference/people-walking.mp4",
-    on_prediction=render_boxes
-)
-
-pipeline.start()
-pipeline.join()
-```
-
-## 🔑 keys
+  <!-- Add more .tutorial-item blocks as needed -->
+</div>
 
-Inference enables the deployment of a wide range of pre-trained and foundational models without an API key. To access thousands of fine-tuned models shared by the [Roboflow Universe](https://universe.roboflow.com/) community, [configure your](https://app.roboflow.com/settings/api) API key.
+<style>
+.button-holder {
+  margin-bottom: 1.5rem;
+}
 
-```bash
-export ROBOFLOW_API_KEY=<YOUR_API_KEY>
-```
+.button {
+  background-color: var(--md-primary-fg-color);
+  display: flex;
+  padding: 10px;
+  color: white !important;
+  border-radius: 5px;
+  text-align: center;
+  align-items: center;
+  justify-content: center;
+}
 
-## 📚 documentation
+.md-typeset h2 {
+  margin-top: 1rem;
+}
 
-Visit our [documentation](/) to explore comprehensive guides, detailed API references, and a wide array of tutorials designed to help you harness the full potential of the Inference package.
+ul {
+  line-height: 1rem;
+}
 
-## © license
+/* Hide <h1> on homepage */
+.md-typeset h1 {
+  display: none;
+}
+.md-main__inner {
+  margin-top: -1rem;
```

Comment:
```suggestion
  margin-top: 1rem;
```

Reply:
This was on purpose; the gap was big. This makes the video align directly w/ the bottom of the nav bar